# Request
Bool just_odom=false
---
# Response
geometry_msgs/Pose2D position
